test/conftest.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from src.play_scraper import PlayPageScraper
+
+
+def pytest_configure():
+    pytest.FILE = "file:"
+
+
+@pytest.fixture(scope="session")
+def icon_dir(tmp_path_factory):
+    return tmp_path_factory.mktemp("icon_dir")
+
+
+@pytest.fixture(scope="session")
+def test_dir(tmp_path_factory):
+    return tmp_path_factory.mktemp("test_dir")
+
+
+@pytest.fixture(scope="module")
```

Comment:
As mentioned above, I would normally use no scope here. The only exception would be if your tests would run unreasonably long, which I don't think they do.